metadata-service/graphql-servlet-impl/src/main/java/com/datahub/graphql/GraphQLController.java

```diff
@@ -36,7 +36,8 @@
 public class GraphQLController {
 
   public GraphQLController() {
-
+    MetricUtils.get().counter(MetricRegistry.name(this.getClass(), "error"));
```

Comment:
Where is the error metric being incremented?

Reply:
L143